### ISSUE TITLE
setVar helper doesn't work in helpers creating new contexts (repeat)

### ISSUE THREAD

**Issue by MoiseScalzo** [mockoon/mockoon#429]
**Is your feature request related to a problem? Please describe.**
In multiple cases I needed a way to use `setVar` helper inside another helper like `#repeat`

**Describe the solution you'd like**
Allow `setVar` to works in a loop and have a new var for each item in the loop.

**Describe alternatives you've considered**
Is maybe better to post an example
```javascript
{
    "inputs": [
        {{# repeat 6 }}
          {{ setVar 'inputType' (oneOf (array 'text' 'checkbox' 'radio' )) }}
            {
              "type": "{{ inputType }}",
              "name": "{{ faker 'git.branch' }}",
              "value": 
                  {{# switch 'inputType' }}
                      {{# case "text" }}"John"{{/ case }}
                      {{# case "textarea" }}"lorem ipsum dolor"{{/ case }}
                      {{# case "checkbox" }}{{ faker 'random.boolean' }}{{/ case }}
                      ...
                  {{/ switch}}
              "label": ""
            }
      {{/ repeat }}
    ]
}
```

**Additional context**
I already tried this kind of template but I got an error `Error while serving the content: Cannot set property 'inputType' of undefined` I supposed this kind of setVar usage is not allowed

**Comment by MoiseScalzo** [comment]
Hi @255kb thank you for the fix. Do you think is possibile, in some way, to have a sort of hotifx in the current version? Or, maybe, build the current version with the new @common package that has the fix?
Thanks

**Comment by 255kb** [comment]
Hi @MoiseScalzo no sorry, it's a desktop app that doesn't support any type of hotfix. This fix will be in the next release v1.13.0 (no ETA but around April). 
You can still build it locally by cloning the repos and following the contribution guidelines.

**Comment by MoiseScalzo** [comment]
> Hi @MoiseScalzo no sorry, it's a desktop app that doesn't support any type of hotfix. This fix will be in the next release v1.13.0 (no ETA but around April).
> You can still build it locally by cloning the repos and following the contribution guidelines.

Thanks for the reply 👍 :)
